docs/administration-guide/secure-database-connections-with-ssl-certificates.md

```diff
@@ -42,7 +42,7 @@ You'll need to specify the location of the certificate on the server that's runn
 
 For example, when connecting to a PostgreSQL database, you'll need to add two parameters:
 
-- `sslmode`. You can see the full list of options in PostgreSQL's documentation. We recommend you use `verify-full`; it's the most secure, and overhead is minimal.
+- `sslmode`. You can see the full list of options in [PostgreSQL's documentation](https://www.postgresql.org/docs/current/libpq-ssl.html). We recommend you use `verify-full`; it's the most secure, and overhead is minimal.
```

Comment:
Wrong library - alternatively this https://jdbc.postgresql.org/documentation/head/connect.html
```suggestion
- `sslmode`. You can see the full list of options in [PostgreSQL's documentation](https://jdbc.postgresql.org/documentation/head/ssl-client.html). We recommend you use `verify-full`; it's the most secure, and overhead is minimal.
```